plugins/commands/migrate/src/main/kotlin/MigrateCommand.kt

```diff
@@ -169,3 +182,12 @@ private fun convertHoconToYaml(hocon: String): String {
 
     return yamlLines.joinToString("\n")
 }
+
+/**
+ * Return the path for the directory to store [PackageConfiguration]s for this [Identifier].
+ *
+ * TODO: Replace with equivalent function from the package configuration provider for the ort-config repository, once
+ *       implemented.
+ */
+fun Identifier.toPackageConfigurationDir() =
```

Comment:
Instead of this function, could you use `Identifier.toPath(emptyValue = "_")`?

Reply:
Isn't this the same as my comment above?

Reply:
It is yes. I haven't seen it. Maybe I didn't hit F5 right before the review.